src/overlays/actors/ovl_En_Guest/z_en_guest.c

```diff
@@ -8,7 +9,20 @@ void EnGuest_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnGuest_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnGuest_Update(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_80A5046C(EnGuest* this);
+void func_80A50518(EnGuest* this, GlobalContext* globalCtx);
+void func_80A5057C(EnGuest* this, GlobalContext* globalCtx);
+void func_80A505CC(Actor* thisx, GlobalContext* globalCtx);
+s32 EnGuest_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* actor);
+void EnGuest_Draw(Actor* thisx, GlobalContext* globalCtx);
```

Comment:
we want to keep the 4 main functions grouped together, with a line break to separate everything else. can look at other actors for an example.
Also you dont need some of these prototypes. we want to just keep the ones that are needed. func_80A5046C and EnGuest_OverrideLimbDraw dont need one.